Makefile

```diff
@@ -48,32 +48,32 @@ setup_githooks:
 build: setup
 	go build -o $(BINARY) $(APP_FILE)
 
-.PHONY: build_linux
-build_linux: setup
+.PHONY: build-linux
+build-linux: setup
 	env GOOS=linux GOARCH=amd64 go build -o $(BINARY_LINUX_64) $(APP_FILE)
 
-.PHONY: docker_build
-docker_build: build_linux
-	docker build -t $(DOCKER_LATEST_TAG) --build-arg BINARY=$(BINARY_LINUX_64) .
+.PHONY: build-image
```

Comment:
@camilamacedo86 do we need this Make command?

Reply:
usually commands use - instead of .. just trying to follow the common standard.